sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/sarif/SarifParser10Test.java

```diff
@@ -226,4 +226,27 @@ public void sarif_version_1_0_secondary_locations_messages() throws IOException
     verifyNoMoreInteractions(callback);
   }
 
+  @Test
+  public void sarif_version_1_0_relative_paths() throws IOException {
+    SarifParserCallback callback = mock(SarifParserCallback.class);
+    new SarifParser10(null, getRoot("v1_0_relative_paths.json"), String::toString).accept(callback);
+    InOrder inOrder = inOrder(callback);
+
+    Location s1144PrimaryLocation = new Location("SourceGeneratorPOC.Generators\\SourceGeneratorPOC.SourceGenerator\\Greetings.cs", "Remove the unused private method 'UnusedMethod'.", 7, 8, 7, 46);
+    inOrder.verify(callback).onIssue("S1144", "warning", s1144PrimaryLocation, new ArrayList<>());
+
+    Location s1186PrimaryLocation = new Location("SourceGeneratorPOC.Generators\\SourceGeneratorPOC.SourceGenerator\\Greetings.cs", "Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.", 7, 28, 7, 40);
+    inOrder.verify(callback).onIssue("S1186", "warning", s1186PrimaryLocation, new ArrayList<>());
+  }
+
+  @Test
+  public void sarif_version_1_0_region_with_length() throws IOException {
```

Comment:
This case, although supported was not covered by unit tests. 

While working on this I've done also a double check to see if the `length` it's documented as a part of the `region` properties in the [json schema](http://json.schemastore.org/sarif-1.0.0) and it is.